tools/topology/CMakeLists.txt

```diff
@@ -76,6 +76,7 @@ set(TPLGS
 	"sof-glk-da7219-kwd\;sof-cml-da7219-max98390\;-DCODEC=MAX98390\;-DPLATFORM=cml"
 	"sof-glk-da7219\;sof-glk-da7219"
 	"sof-glk-rt5682\;sof-glk-rt5682"
+	"sof-glk-cs42l42\;sof-glk-cs42l42"
```

Comment:
looking at the commit message, could we use just one fine with two macros for the dailink and bitclock delay? Topologies are hard to maintain, and if we can factor it's best.

Reply:
sure, will avoid adding new topology file.